### ISSUE TITLE
[1.10.2] chiselsandbits

### ISSUE THREAD

**Issue by FiveEYZ** [SpongePowered/SpongeForge#903]
Chisels and bits isn't working. I can't chisel anything, nor place any bits.

This error generates while doing whats said above:
[latest.log](https://gist.github.com/FiveEYZ/3ab7e3fb5f6e6531afa0560cb8eca3e1)
[fml-server-latest.log](https://gist.github.com/FiveEYZ/01ef0cc97cbb2a77e8031ae117cca6ba)

using:
Minecraft: 1.10.2
SpongeAPI: 5.0.0-SNAPSHOT-6211516
SpongeForge: 1.10.2-2079-5.0.0-BETA-1727
Minecraft Forge: 12.18.1.2086 also tested on 2.18.1.2079
chiselsandbits-12.1

**Comment by Aaron1011** [comment]
@FiveEYZ: I'm unable to reproduce this. Can you try again with the latest SpongeForge, and list the exact steps you took to cause this error?

**Comment by FiveEYZ** [comment]
Tested with SpongeForge 1731 and forge 2079 / 2088.

1.  Start the server (no special java flags)
2.  OP myself
3.  Put myself in gamemode 1
4.  Take any chisel tool (I chose a Stone Chisel)
5.  Put myself in gamemode 0
6.  De-OP myself
7.  Left-click anywhere on any block

These steps might seem simple in mind, but this is really the only things I do. 
The mod works without sponge added, and it recently started to happen (guess i can backtrack that).

Here are new logs:
[fml-server-latest.log](https://gist.github.com/FiveEYZ/a61983b213d697b67a93744914ac172d)
[latest.log](https://gist.github.com/FiveEYZ/08647e1fd7ef43abb8c8c4f1db1c63e0)
